src/main/java/fact/cleaning/TwoLevelTimeMedian.java

```diff
@@ -55,10 +53,10 @@
     public int minNumberOfPixel;
 
     @Parameter(required = false)
-    public String[] starPositionKeys = null;
+    public String starPositionsKey = null;
```

Comment:
does this change the meaning of this key? Could you add a doc string?

Reply:
done